src/types.ts

```diff
@@ -115,8 +122,8 @@ export interface SavedGroup {
 export interface User {
   _id: string; // mongoose ObjectId
   firebaseUid: string;
-  imgUrlProfileSmall?: string;
-  imgUrlProfileLarge?: string;
+  imgUrlProfileSmall: string;
+  imgUrlProfileLarge: string;
```

Comment:
why did we make this required?

Reply:
iirc users can choose not to upload profile pic. 
unless our backend populates this with an empty string (which I don't think it does?)

Reply:
uh idk idr doing that